x/staking/keeper/keeper.go

```diff
@@ -93,6 +93,16 @@ type Keeper struct {
 	LastValidatorPower collections.Map[[]byte, gogotypes.Int64Value]
 	// Params key: ParamsKeyPrefix | value: Params
 	Params collections.Item[types.Params]
+	// ValidatorConsPubKeyRotationHistory: consPubkey rotation history by validator
+	ValidatorConsPubKeyRotationHistory collections.Map[collections.Pair[[]byte, uint64], types.ConsPubKeyRotationHistory]
```

Comment:
`ConsPubKeyRotationHistory` contains a `Fee`, are we ok storing it?

Reply:
Yes, that fee holds the fee used for rotation